docs/test.ts

```diff
@@ -536,3 +536,100 @@ const otelTraceId: string = spanContext.traceId
 const otelSpanId: string = spanContext.spanId
 const otelTraceFlags: number = spanContext.traceFlags
 const otelTraceState: opentelemetry.TraceState = spanContext.traceState!
+
+// -- LLM Observability --
+const llmobsEnableOptions = {
+  apiKey: 'dd-api-key',
+  mlApp: 'mlApp',
+  agentlessEnabled: true
+}
+tracer.init({
+  llmobs: llmobsEnableOptions,
+  experimental: {
+    llmobs: llmobsEnableOptions
+  }
+})
+const llmobs = tracer.llmobs
+const enabled = llmobs.enabled
+
+// manually enable
+llmobs.enable({
+  apiKey: 'dd-api-key',
+  mlApp: 'mlApp',
+  agentlessEnabled: true
+})
+
+// manually disable
+llmobs.disable()
+
+// trace block of code
+llmobs.trace({ name: 'name', kind: 'llm' }, () => {})
+llmobs.trace({ kind: 'llm', name: 'myLLM', modelName: 'myModel', modelProvider: 'myProvider' }, () => {})
```

Comment:
Just a note: model name and provider are no longer required for LLM spans, if not provided they should default to `custom` and `custom` respectively. Does this PR reflect that?

Reply:
yes! here's the [condition](https://github.com/DataDog/dd-trace-js/blob/7c7a328b2f52da0b9b515ca5d32e1af23bdf3097/packages/dd-trace/src/llmobs/span_processor.js#L73-L76) in the span processor that reinforces that. additionally, the [span options type definition](https://github.com/DataDog/dd-trace-js/blob/7c7a328b2f52da0b9b515ca5d32e1af23bdf3097/index.d.ts#L2471-L2498) list both as optional (`?`) as well